.github/workflows/nightly-check-ci.yml

```diff
@@ -11,18 +11,31 @@ on:
 
 jobs:
   nightly:
+    name: nightly (${{ matrix.gradle-task }}, Java ${{ matrix.test-jvm-version }}, Jetty ${{ matrix.jetty-version }})
     strategy:
       fail-fast: false
       matrix:
         gradle-task: ['check', 'testSerial', 'testParallel', 'testOutOfBand']
-        test-jvm-version: ['11', '17', '21', '23']
+        test-jvm-version: ['17', '21', '23']
+        jetty-version: ['12']
+        # While the defaults above only run client integration tests on Jetty 12, the following extra rules restore
+        # those tests for Jetty 11. Additionally, restores the Java 11 test version, but only when using Jetty 11, to
+        # avoid duplication. Does not run Jetty 11 tests for all JDKs.
+        include:
+          - gradle-task: 'check'
+            test-jvm-version: '11'
+            jetty-version: '11'
+          - gradle-task: 'check'
+            test-jvm-version: '17'
+            jetty-version: '11'
     if: ${{ github.repository_owner == 'deephaven' || github.event_name != 'schedule' }}
     runs-on: ubuntu-24.04
     concurrency:
-      group: ${{ matrix.gradle-task }}-${{ matrix.test-jvm-version }}-${{ github.workflow }}-${{ github.ref }}
+      group: ${{ matrix.gradle-task }}-${{ matrix.test-jvm-version }}-${{ matrix.jetty-version }}-${{ github.workflow }}-${{ github.ref }}
       cancel-in-progress: true
     env:
       COVER: ${{ (github.event_name == 'schedule' || startsWith(github.ref_name, 'coverage/')) && matrix.test-jvm-version == '21' }}
```

Comment:
Even though this isn't getting triggered right now since we aren't doing test-jvm-version 21 w/ jetty-version 11, the intention was to limit coverage to a entry for each gradle-task. As such, I think we should additonally qualify this w/ `matrix.jetty-version == '12'`